src/database/seeds/004_medico.js

```diff
@@ -0,0 +1,38 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('medico').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('medico').insert([
+        {
+          id_usuario: 4,
+          crm: "123456789",
+          especialidade: "Dermatologista",
+          created_at: null, 
+          updated_at: null 
```

Comment:
Deletar estas linhas caso você concorde em retirar esses atributos da tabela Medico.

```suggestion
          crm: "123456789"
```